node/src/main.rs

```diff
@@ -391,75 +402,70 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
         logger.clone(),
         Arc::new(subgraph_provider),
         store.clone(),
+        node_id.clone(),
     ));
+    named_subgraph_provider
+        .start()
```

Comment:
Why must the rest of `main` wait on this, rather than starting it in parallel with `tokio::spawn`?

Reply:
hmmm OK yeah I guess that's safe

Reply:
done!